mobile/src/helpers/strings/word-capitalize/word-capitalize.ts

```diff
@@ -0,0 +1,7 @@
+const wordCapitalize = (word: string): string => {
```

Comment:
Rename it to just capitalize or capitalizeFirstLetter if you want, and we don't need such modifications, just `${string[0].toUpperCase()}${string.slice(1)}` without calling lowercase